src/styles/pages/docs.scss

```diff
@@ -4,6 +4,12 @@
 .docs {
 	display: flex;
 	background-color: var(--fds-solid-background-base);
+
+	.markdown-body {
+		font-size: 1.6rem;
+		line-height: 2.6rem;
+		max-inline-size: 1000px;
```

Comment:
why this size specifically?

Reply:
I initially used the same as the blog post but it was too small.